src/pilot/config_dashboard.py

```diff
@@ -703,6 +710,22 @@ def prep_subscription_json(subUser, subPass, physId, cephId):
     os.unlink(tmp_file)
 
 
+def prep_subscription_json_satellite(satOrg, satKey, physId, cephId):
+    # Prepares the subscription.json file
+    LOG.info("Preparing the subscription json file.")
+
+    tmp_file = os.path.join("/tmp", "subscription.json-mod")
+    os.system('cp ' + Node.subscription_json + ' ' + tmp_file)
+    sed_inplace(tmp_file, "_satellite_credentials", "satellite_credentials")
+    sed_inplace(tmp_file, "CHANGEME_organiaztion", satOrg)
```

Comment:
Nit: probably should change CHANGEME_organiaztion to CHANGEME_organization in the line above